src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+/* Stars Start */
+.stars {
+  display: flex;
+}
+
+.stars__item {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background: url(../images/star.svg) no-repeat center;
+}
+
+.stars:last-child {
+  margin-right: 0;
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
we should have the opportunity to change nr of start just by changing the modifier